### ISSUE TITLE
What kind of Group is for what kind of work?

### ISSUE THREAD

**Issue by frivoal** [w3c/w3process#804]
What kind of Group is for what kind of work? If you want to deliver REC track or Registry track documents, you need a Working Group.

Beyond, that, things are pretty fuzzy. There are a bunch of differences between WGs, IGs, CGs, and BGs, but it's not clear to me that they really tell us which kind of group is suitable for which purpose.

In response to a recent AC review for a charter, a member felt ["This IG should instead be a CG"](https://lists.w3.org/Archives/Public/public-new-work/2023Dec/0000.html) about a proposed group. It's not clear to me that the Process contains much to either to support or refute that claim.

Here are some questions I'm not sure we have an answer to:
* When a group's sole output is reviews of what others do, should it be a WG? an IG? a CG? (See also https://github.com/w3c/w3process/issues/414)
* When a group's sole output is Notes, should it be a WG or an IG? Also the only real rule about Notes is that they need the consensus of the group that published them. Nothing stands in the way of CGs doing that too, even if they cannot technically call such documents Notes. But only WGs and IGs can elevate such documents to Statements.
* IGs can be chartered to have [open public participation](https://www.w3.org/2023/Process-20231103/#ig-mail-only). When is it preferable to have one of these over a CG, or the other way around.

Here are some other assorted differences:
* WGs have strong patent coverage. CGs have some patent coverage. IGs have none.
* WGs and IGs are subject to the Process, and therefore to Formal Objections, the Council, and so on. CGs have their own separte (lightweight) process, which doesn't include those concepts.
* WGs and IGs need an AC-Approved charter, CGs do not (they can have a charter, but don't need to, and if they do have one, it doesn't need AC approval).
* You need to be a Member (or an Invited Expert) to be part of a WG (though that's not needed to submit feedback). The same is true for some IGs, though other IGs can be open to anyone. Anyone can join a CG.
* CG may—but do not need to—use consensus as the basis of decision making. WGs and IGs have to.
* I could not find a requirement to abide by the CEPC in the texts people need to agree to in order to join a CG. (This surprised me. Hopefully I just missed it)
* I could not find a description of disciplinary procedures for CGs. (This surprised me. Hopefully I just missed it)
* Both CGs and IGs/WGs have requirements about archival, but those requirement are not identical.

Oh, and we have Business Groups too. I suspect the goal for those is somewhat clearer, but then again maybe not, and there could be some interesting overlap with IGs in some cases.

I think we should spend some time thinking about:
* Do all these types of groups serve distinct purposes? Should we combine some?
* Are the differences between these types of groups serving those distinct purposes, or are some of them accidental/historical? Should we harmonize more?
* Would incorporating CGs and BGs into the process, as asked in https://github.com/w3c/w3process/issues/326 or https://github.com/w3c/w3process/issues/409 help harmonize them with WGs and IGs, or would that make the unified Process needlessly complex?
* When some goals can be fulfilled by multiple types of groups, should people be free to pick whichever type they prefer? Should we require things like "Don't charter a WG if an IG could do it" / "Don't charter an IG if a CG could do it"?

**Comment by michaelchampion** [comment]
I agree it's time for the Process to explicitly discuss CGs / BGs, and clarify their relationship to IGs and WGs. Some thoughts off the top of my head:

- I'm unconvinced that BGs have any reason to exist at this point.  Use cases for a "CG with Team support" could be covered by an IG open to non-member participation AFAICT.  
- I'd like to see CGs explicitly mentioned in the Process: CG's start with a Problem; WG's start with a rough outline of a solution to that problem.  Likewise the Process should point to CGs as the appropriate mechanism to "incubate" specs: Build a community of people with a shared challenge who develop a technology to address it and a spec for it's interoperability features (API, protocol, format ...) to the "proof of concept" stage where there is a critical mass of users and implementers who agree it's ready to standardize.  
- CGs MUST abide by the CoC/CEPC and work by some sort of defined consensus-seeking process (although not necessarily all the constraints and details defined in  the Process Document).  For example, CGs have no ability to appeal to the Team or a FO Council if they can't get consensus.  CGs can be test beds for process innovations (WHATWG-style strong editors, chairs empowered to make decisions even in the face of dissent, various voting mechanisms, whatever) ... but within the broad constraint of having a defined consensus-seeking process.
- IGs can continue more or less as presently defined to focus on understanding a problem, doing "gap analysis" to document current standards' lack of ability to address a problem, and to build a community. IGs should not incubate solutions (their patent policy doesn't support that). But successful IGs shouldn't immediately charter a WG to find a solution, they should spin off one or more CGs to incubate solutions, and propose WG only when they have reached a level of maturity.  
- The model of parallel CGs for incubation and WGs for standardization can work well, and the Process should mention that (and perhaps offer best practice guidance derived from experience with groups such as Web Assembly that succeed with it).
- It should be HARD to charter a WG, that is a feature not a bug.  If competing CGs, or different factions in a CG, have very different visions for how to address a problem or disagree on fundamental aspects of a solution, W3C should not create a WG to sort it all out.  The competing communities need to find common ground -- or the marketplace pick a winner -- before W3C creates a WG.  The WG process is too heavyweight and burdensome on the AC and FO Councils to use to pick fundamental technical directions.

**Comment by michaelchampion** [comment]
Also, I'm not sure if this is a separate issue or not, but I think it would clarify the Process and the "what type of group?" question to ground it in Problem Statements:

- Creating a CG  is fairly easy, it just requires a statement of a problem (or unrealized opportunity) with the current web platform / ecosystem and a suggestion to build a community to explore the problem and possible solutions.  Participants agree to abide by the CoC/CEPC, work by some definition of "consensus", and to make RF patent commitment on their own contributions to the discussion. No explicit approval by the Team or other W3C bodies is needed for the CG to start working.
- Creating a WG is quite it bit harder. There should be a clear problem statement, evidence a community collectively addressing the problem, an incubated technical spec describing the broad outline of a solution, and a critical mass of platform implementers / website builders / users who plan to implement/deploy/use that solution once it is mature. Participants agree to work according to the W3C Process and make the commitments required by the Patent Policy.  Approval of a WG charter requires consensus from the W3C membership that the WG would pursue a significant problem and there is a well-functioning community working together to solve it.
- Getting a Recommendation is quite difficult ...  The WB must have created and tested a spec that is defined clearly enough to be interoperably implemented in all its details, there must be a real-world ecosystem of implementers and users that implement and use the spec, and there must be solid empirical evidence that **implementations of the spec really  improve the situation described in the CG->WG problem statement**.   Approval of a Recommendation requires strong and broad consensus from the W3C membership that the spec meets these criteria.

**Comment by brentzundel** [comment]
closing based on CG discussion today
